transforms/language/lang_id/ray/src/lang_id_transform_ray.py

```diff
@@ -27,10 +27,14 @@
 model_kind_key = "model_kind"
 model_url_key = "model_url"
 content_column_name_key = "content_column_name"
+output_lang_column_name_key = "output_lang_column_name"
+output_score_column_name_key = "output_score_column_name"
```

Comment:
Why are you redefining these here?  They should be the same as in the python transform right?  If so, use them instead.

Reply:
Remember that this ray-runtime deployment of the python transform depends on the adjacent python source and is installed both in the venv and the docker image.

Reply:
I completely agree with your comment.
I just followed the pattern in noop. I thought that's the code conventions.